user/physlock/patches/physlock-13-Improved-commandline-help.patch

```diff
@@ -0,0 +1,89 @@
+From e6c69762f5f555fa17659e3440d8392b6a47bbc6 Mon Sep 17 00:00:00 2001
```

Comment:
do we _really_ need this patch? no strong opinion but i think it should be removed still